<launch>
	<node pkg="RDP"
		  type="MotionCapture"
		  name="capture"
		  output="screen"
		  respawn="false"
    />
	<node pkg="RDP"
	      type="UserRecognition"
		  name="userrecognition"
		  output="screen"
		  respawn="false"
    />
</launch>

